frame/staking/src/tests.rs

```diff
@@ -3363,10 +3378,13 @@ fn test_payout_stakers() {
 			Staking::reward_by_ids(vec![(11, 1)]);
 
 			// compute and ensure the reward amount is greater than zero.
-			let _ = current_total_payout_for_duration(reward_time_per_era());
+			let payout = current_total_payout_for_duration(reward_time_per_era());
+			let actual_paid_out = payout_exposure_part.mul_ceil(payout);
```

Comment:
I am not sure why only `.mul_ceil` worked here; `.mul` ends up being 1 less which I don't understand because we use normal multiplication here: 

https://github.com/paritytech/substrate/blob/f72fe056844a375f16449555d6e20e57dcd16aa6/frame/staking/src/pallet/impls.rs#L186-L187

Reply:
I feel we should figure this out before we merge :/

Reply:
[002dafc](https://github.com/paritytech/substrate/pull/10887/commits/002dafc33e47bc1900d8b3d276de251973d2aba7)